recipes/flask-openid/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{%set name = "Flask-OpenID" %}
+{%set version = "1.2.5" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "5a8ffe1c8c0ad1cc1f5030e1223ea27f8861ee0215a2a58a528cc61379e5ccab" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - flask >=0.3  # [py2k]
+    - flask >=0.10.1  # [py3k]
```

Comment:
I have no idea, but that's what it says in [setup.py](https://github.com/mitsuhiko/flask-openid/blob/master/setup.py)